inst/qml/Descriptives.qml

```diff
@@ -75,6 +77,26 @@ Form
 			title:	qsTr("Quantiles")
 			info:	qsTr("Percentile Values")
 
+			DropDown
+			{
+				name: "quantilesType"
+				label: qsTr("Type")
+				id: quantilesType	
+				indexDefaultValue: 6 // Type 7, the default in R 
+				info: qsTr("Method used to compute quantiles (see Hyndman & Fan, 1996; Langford, 2006, for more information). The selection carries over to the inter-quartile range (IQR), box- and QQ-plot calculations. Note that ordinal variables are treated as continuous for the computations.\n")
```

Comment:
Perhaps also mention which values corresponds to other software programs? because the help file will be generated from this if I remember correctly, so it might be nice to mention this there as well.